onboarding/forms.py

```diff
@@ -16,12 +16,12 @@
 
 
 # Whenever we change the fields in any of the onboarding
-# forms, we should change this number to ensure that we
+# forms, we should increment this number to ensure that we
 # never use an old session's onboarding data with the
 # new validation logic. The downside is that the old
 # session's onboarding data will disappear, but hopefully
 # we won't have to do this often.
-FIELD_SCHEMA_VERSION = 4
+FIELD_SCHEMA_VERSION = 5
```

Comment:
Hmm, it's possible that we don't need to increment this now that we are manually migrating the old schema to the new one (see the `migrate_dict` method in this PR)...  Which I thought was the reason we added that auto-migration functionality (otherwise incrementing this number means that people who are in the middle of onboarding will suddenly have all their onboarding data cleared, which would be confusing).